docs/release-notes/february-2023.md

```diff
@@ -0,0 +1,71 @@
+---
+slug: february-2023
+append_help_link: true
+hide_title: true
+description: >-
+  Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+---
+
+# February 2023
+
+## Important update
+
+- Semgrep CLI is now officially renamed to **Semgrep OSS Engine**. Note that Semgrep CLI is an interface, not a product. As an interface, you can use the CLI as an interface for several products. For example the OSS, Semgrep Code, Semgrep Supply Chain.
+- Team tier rules are now renamed to Pro Rules. The **Pro rules** are created by r2c and targeted for security and software engineers who need accurate findings. These rules were previously called Team tier rules. As of this update, these rules are officially called the **[Pro rules](/semgrep-code/pro-rules/)** and are available with the [Team or higher tier](https://semgrep.dev/pricing).
+- DeepSemgrep is renamed to **Semgrep Pro Engine**. Semgrep Pro Engine is fully available for [Team or higher tier](https://semgrep.dev/pricing) users. See [Semgrep Pro](/deepsemgrep/deepsemgrep-introduction/) documentation. Some specific flags have been removed:
```

Comment:
nitpick: **Semgrep Pro Engine** contains more than just DeepSemgrep.
So maybe we should rephrase it to "DeepSemgrep has been merged into **Semgrep Pro Engine**"
But that's really a nitpick

Reply:
@bmahe more such nitpicks, please! Thank you. :)

Reply:
```suggestion
- DeepSemgrep has been bundled with other functionalities under the name of **Semgrep Pro Engine**. Semgrep Pro Engine is fully available for [Team or higher tier](https://semgrep.dev/pricing) users. See [Semgrep Pro](/deepsemgrep/deepsemgrep-introduction/) documentation. Some specific flags have been removed:
```